release/__init__.py

```diff
@@ -298,11 +299,8 @@ def make_stable_artifacts(cache_repository_url):
 
     # TODO(cmaloney): Rather than guessing / reverse-engineering all these paths
     # have do_build_packages get them directly from pkgpanda
-    try:
+    with logger.scope("Building packages"):
         all_completes = do_build_packages(cache_repository_url)
-    except pkgpanda.build.BuildError as ex:
-        print("ERROR Building packages:", ex, file=sys.stderr)
```

Comment:
it's critical we still catch / pretty-print the error message here. Changes usability quite a bit.